src/_guides/language/language-tour.md

```diff
@@ -3014,9 +3002,9 @@ class Television {
   // ···
 }
 
-class SmartTelevision extends Television {
+class SmartTelevision [!extends!] Television {
   void turnOn() {
-    super.turnOn();
+    [!super!].turnOn();
```

Comment:
I had started adding some extra highlighting to some of the examples, but stopped. We can add more in a separate pass.

When the code excerpt is non trivial (like in this case), judiciously placed highlights help focus the eye of the reader.